website/src/components/NotificationBanner/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link, makeStyles } from '@material-ui/core';
+import notifications from '../../resources/notifications.json';
+
+interface Notification {
+  message: string;
+  url?: string;
+  linktext?: string;
+}
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Can we move CSS to style.ts?

Reply:
But since we follow this method of separating out CSS in a file. I think it will be better to keep it consistent